rasa/cli/utils.py

```diff
@@ -4,6 +4,8 @@
 import sys
 from typing import Any, Dict, List, Optional, TYPE_CHECKING, Text
 
+from rasa.shared.utils.cli import print_error
```

Comment:
Let's not import this directly but rather by module. Doing this can lead to the case which I'm fixing here https://github.com/RasaHQ/rasa/pull/6646